### PR TITLE
Upgrade aea code to 1.1.1 and update solc

### PR DESCRIPTION
**Pull request by danielksan81**
- Upgraded config and dependencies from AEA to current version 1.1.1
- Updated fingerprints inaea configs
- Updated solc to use v0.8.4
- Changed default port for ethereum in truffle to 8545

Signed-off-by: Daniel Kunz <daniel.kunz@de.bosch.com>

### CODE REVIEW

**Comment by danielksan81** [comment]
resolves #16

File: EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt [hunk 1]
```diff
@@ -1 +1 @@
-e6e8981d75dc59538d78bdaeed0cfb7b1321358beeec415109ed54455ec9f024
\ No newline at end of file
+0x8f1bb6383b67a208bd73c2bc146167ef468d6a74d2e172d9c5b374a9d255c514
```
Comment (l-wegner) [inline EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt:1]: Shall we create ticket to remove the private keys from the repo and provide documenation how to generate them conveniently?

**Review by l-wegner — APPROVED** [review]
LGTM

I skipped the contents of `**/vendor/**`, because this pulled code.
I also skipped .json file put out by solc.

File: EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt (hunk 1 above)
Comment (danielksan81) [reply EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt:1]: having some defined accounts makes it easier for https://github.com/boschresearch/open-eot-agents/issues/2 to set up a system with just a few commands. Therefore I think it is needed to have some predefined.
At least in the documentation all proper configurations need to be mentioned for adoption.
What do you think?

Comment (l-wegner) [reply EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt:1]: Having private keys in the code base bears the risk, some adopter forgets to recreate their own, when creating something for production. I think right now it's no issue, but maybe for the future.

**Pull request merged**